uui/components/widgets/Tag.module.scss

```diff
@@ -67,75 +67,74 @@
     &.size-18 {
         min-height: 18px;
         min-width: 18px;
-        padding-left: 3px;
-        padding-right: 3px;
+        padding: 2px;
 
         :global(.uui-count_indicator):not(:last-child) {
             margin-right: 3px;
         }
 
         :global(.uui-caption), :global(.uui-input) {
-            padding: 2px 3px;
+            padding: 0 3px;
             line-height: 12px;
             font-size: 12px;
         }
     }
```

Comment:
Why you change vertical layout in all sizes?

Reply:
checked and set optimal sizes